src/Stakeholders/components/FocalPeople/List/index.js

```diff
@@ -211,7 +211,7 @@ class FocalPersonsList extends Component {
               },
               () => {
                 notifyError(
-                  'An Error occurred while refreshing Focal People please focalPerson system administrator'
+                  'An Error occurred while refreshing Focal People please focalPerson co system administrator'
```

Comment:
Check the grammar here.

Reply:
This have being fixed in the previos PR for focal people

Reply:
Cool.